webapp/tests/mocha/unit/purger.spec.js

```diff
@@ -64,21 +64,31 @@ describe('Purger', () => {
       });
     });
 
-    it('should throw fetch errors', () => {
-      fetch.rejects({ some: 'error' });
-      return purger.info().catch(err => {
-        chai.expect(err).to.deep.equal({ some: 'error' });
-      });
+    it('should throw fetch errors', done => {
+      fetch.resolves({ json: sinon.stub().resolves({ code: 500, error: 'Server error' }) });
+      purger.info()
+        .then(() => {
+          done(new Error('Expected error to be thrown'));
+        })
+        .catch(err => {
+          chai.expect(err.message).to.equal('Error fetching purge data: {"code":500,"error":"Server error"}');
+          done();
+        });
     });
   });
 
   describe('checkpoint', () => {
-    it('should throw fetch errors', () => {
```

Comment:
Could we keep this test and just add the new one where the server returns 500? Fetch throws when not connected, which is much more likely to occur frequently than API failing.

Reply:
Absolutely. Done.